products/idn/tools/cli/index.md

```diff
@@ -0,0 +1,224 @@
+
+### Assisted configuration
+
+Once you have created the authentication method, you can use the `configure` command for your chosen method to configure the CLI for your tenant. 
+
+This command creates a configuration file in your home directory to store your tenant's connection details.
+
+If you are using a **PAT**, run this command: 
+
+```shell
+sail configure pat
+```
+
+[configure PAT](./assets/img/vhs/configure/configure-pat.gif)
+
+If you are using an **OAuth client**, run this command: 
+
+```shell
+sail configure oauth
+```
+
+[configure OAuth](./assets/img/vhs/configure/configure-oauth.gif)
+
+### Manual configuration
+
+Alternatively, you can manually create a configuration file in your home
+directory.
+
+On **Linux and MacOS**, open your terminal app and run these commands: 
+
+```shell
+mkdir ~/.sailpoint
+touch ~/.sailpoint/config.yaml
+```
+
+On **Windows**, run Powershell as an administrator and run these commands:
+
+```powershell
+New-Item -ItemType Directory -Path 'C:\Users\<username>\.sailpoint'
+New-Item -ItemType File -Path 'C:\Users\<username>\.sailpoint\config.yaml'
+```
+These commands will create a `config.yaml` file you can use to set your environment variables:
+
+The `config.yaml` file must contain the following information:
+
+```yaml
+authtype: pat # Can be either pat or oauth, both methods can be configured, with only one being active at a time.
+debug: false # Setting debug to true, will result in more verbose output
+oauth: # All OAuth specific configuration information
+  authurl: https://example.identitynow.com/oauth/authorize
+  baseurl: https://example.api.identitynow.com
+  clientid: example-client-id
+  clientsecret: ""
+  redirect:
+    path: /callback
+    port: 3000
+  tenant: example
+  token:
+    accesstoken: example-access-token
+    expiry: example-token-expiry-date
+  tokenurl: https://example.api.identitynow.com/oauth/token
+pat: # All Personal Access Token specific configuration information
+  tenant: example
+  baseurl: https://example.api.identitynow.com
+  tokenurl: https://example.api.identitynow.com/oauth/token
+  clientsecret: example-client-secret
+  clientid: example-client-id
+  token:
+    accesstoken: example-access-token
+    expiry: example-token-expiry-date
+```
+You can copy the example into your `config.yaml` file. You must specify your baseurl, tokenurl, clientsecret, and clientid and any other necessary information for your chosen authentication method. 
+
+You can have both authentication methods configured at once, but only one can be active at a time. 
+
+#### Environment variable configuration 
+
+You can also store your configuration in environment variables. 
+On **Linux/Mac**, open your terminal app and run these commands to export the following environment variables:
+
+```shell
+export SAIL_BASEURL=https://{tenant}.api.identitynow.com
```

Comment:
The ENV VARS for each of these sections have changed to snake case, examples should be on the main readme of the CLI

Reply:
Okay I updated these to reflect the updates in the readme.